pkg/asset/asset.go

```diff
@@ -13,45 +13,50 @@ import (
-	AssetPathKubeConfig              = "auth/kubeconfig"
-	AssetPathManifests               = "manifests"
-	AssetPathKubelet                 = "manifests/kubelet.yaml"
-	AssetPathProxy                   = "manifests/kube-proxy.yaml"
-	AssetPathKubeFlannel             = "manifests/kube-flannel.yaml"
-	AssetPathKubeFlannelCfg          = "manifests/kube-flannel-cfg.yaml"
-	AssetPathAPIServerSecret         = "manifests/kube-apiserver-secret.yaml"
-	AssetPathAPIServer               = "manifests/kube-apiserver.yaml"
-	AssetPathControllerManager       = "manifests/kube-controller-manager.yaml"
-	AssetPathControllerManagerSecret = "manifests/kube-controller-manager-secret.yaml"
-	AssetPathScheduler               = "manifests/kube-scheduler.yaml"
-	AssetPathKubeDNSDeployment       = "manifests/kube-dns-deployment.yaml"
-	AssetPathKubeDNSSvc              = "manifests/kube-dns-svc.yaml"
-	AssetPathSystemNamespace         = "manifests/kube-system-ns.yaml"
-	AssetPathCheckpointer            = "manifests/pod-checkpoint-installer.yaml"
-	AssetPathEtcdOperator            = "manifests/etcd-operator.yaml"
-	AssetPathEtcdSvc                 = "manifests/etcd-service.yaml"
+	AssetPathCAKey                       = "tls/ca.key"
+	AssetPathCACert                      = "tls/ca.crt"
+	AssetPathAPIServerKey                = "tls/apiserver.key"
+	AssetPathAPIServerCert               = "tls/apiserver.crt"
+	AssetPathServiceAccountPrivKey       = "tls/service-account.key"
+	AssetPathServiceAccountPubKey        = "tls/service-account.pub"
+	AssetPathAdminKey                    = "tls/admin.key"
+	AssetPathAdminCert                   = "tls/admin.crt"
+	AssetPathAdminKubeConfig             = "auth/admin-kubeconfig"
+	AssetPathBootstrapKubeConfig         = "auth/bootstrap-kubeconfig"
+	AssetPathBootstrapAuthToken          = "auth/bootstrap-auth-token"
+	AssetPathManifests                   = "manifests"
+	AssetPathKubelet                     = "manifests/kubelet.yaml"
+	AssetPathKubeletBootstrapRoleBinding = "manifests/kubelet-bootstrap-role-binding.yaml"
+	AssetPathKubeSystemSARoleBinding     = "manifests/kube-sa-role-binding.yaml"
+	AssetPathProxy                       = "manifests/kube-proxy.yaml"
+	AssetPathKubeFlannel                 = "manifests/kube-flannel.yaml"
+	AssetPathKubeFlannelCfg              = "manifests/kube-flannel-cfg.yaml"
+	AssetPathAPIServerSecret             = "manifests/kube-apiserver-secret.yaml"
+	AssetPathAPIServer                   = "manifests/kube-apiserver.yaml"
+	AssetPathControllerManager           = "manifests/kube-controller-manager.yaml"
+	AssetPathControllerManagerSecret     = "manifests/kube-controller-manager-secret.yaml"
+	AssetPathScheduler                   = "manifests/kube-scheduler.yaml"
+	AssetPathKubeDNSDeployment           = "manifests/kube-dns-deployment.yaml"
+	AssetPathKubeDNSSvc                  = "manifests/kube-dns-svc.yaml"
+	AssetPathSystemNamespace             = "manifests/kube-system-ns.yaml"
```

Comment:
nit: I know not your change, but I don't believe this is used anymore if you wanted to drop it.